docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/charts/library/rms-funding.js

```diff
@@ -191,8 +166,8 @@ function initChart(Chart) {
           id: '2019I',
           xAxis: 1,
           data: [
-            ['Intramural Research', 964.90],
-            ['RMS', 449.89]
+            ['Intramural Research', 964.9],
```

Comment:
Change of precision here.  Just making sure this was intentional.

Reply:
Hi @arcepaul - Yes, that was the intended change! Thanks for checking!